test/python/linear_attention/test_fla_mlp_shim_unit.py

```
@@ -0,0 +1,435 @@
+    module.GatedMLP = GatedMLP
+    return module
+
+
+def test_public_mlp_activation_rejects_unsupported_fla_version(monkeypatch):
+    module = _fake_fla_mlp_module()
+    original = module.GatedMLP.forward
+    monkeypatch.setitem(sys.modules, module.__name__, module)
+    monkeypatch.setattr(fla_api, "_supports_installed_fla", lambda: False)
+    monkeypatch.setattr(fla_api, "_ORIGINALS", {})
+
+    with pytest.raises(ImportError, match="requires flash-linear-attention==0.5.2"):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)